jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/RegexPathSpec.java

```diff
@@ -18,25 +18,29 @@
 
 package org.eclipse.jetty.http.pathmap;
 
+import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class RegexPathSpec extends AbstractPathSpec
 {
-    private final String _declaration;
     private final PathSpecGroup _group;
     private final int _pathDepth;
     private final int _specLength;
     private final Pattern _pattern;
 
+    private static String normalize(String regex)
```

Comment:
Looking below at ServletPathSpec, should this method be called "santiize" rather than normalize? It's not prefixing "/" for example ....